test/plugin/base_test.rb

```diff
@@ -1160,11 +1342,27 @@ def setup_prometheus
     Prometheus::Client.registry.instance_variable_set('@metrics', {})
   end
 
+  def url_with_local_resource_id(local_resource_id)
```

Comment:
Do we want to put a `# Metadata Agent.` comment above these functions?

Reply:
Added.